package.json

```diff
@@ -26,21 +27,23 @@
     "url": "https://github.com/tommyip/bootstrap-icons-vue/issues"
   },
   "scripts": {
-    "build": "node ./src/codegen.js && rollup -c",
+    "build": "node ./src/codegen.cjs && rollup -c",
```

Comment:
Is there a need to add `"type": "module",` and `cjs`?

Reply:
Not sure, i had some issues with the migration to typescript 5 and rollup, could be a change that is not needed.